package.json

```diff
@@ -68,6 +68,7 @@
     "figures": "~2.0.0",
     "first-run": "~2.0.0",
     "identity-obj-proxy": "~3.0.0",
+    "intl-locales-supported": "~1.8.12",
```

Comment:
What does this dependency do, and why does it belong in the root `package.json`?